src/transformers/training_args.py

```diff
@@ -1316,6 +1316,18 @@ class TrainingArguments:
             "help": "Activates neftune noise embeddings into the model. NEFTune has been proven to drastically improve model performances for instrcution fine-tuning. Check out the original paper here: https://arxiv.org/abs/2310.05914 and the original code here: https://github.com/neelsjain/NEFTune. Only supported for `PreTrainedModel` and `PeftModel` classes."
         },
     )
+    
+    deepspeed_force_lr_scheduler_checkpointing: bool = field(
```

Comment:
this isn;t required. Trainer saves the scheduler when it isn't part of DeepSpeed Engine. 
Below is a screenshot of a checkpoint saved with the sceduler file. So, this isn't required.

![Screenshot 2024-02-22 at 4 17 27 PM](https://github.com/huggingface/transformers/assets/13534540/3d1c939d-b30b-4dd0-866b-cd8213bf35e7)